DragonSlayer.cpp

```diff
@@ -1,21 +1,36 @@
 #include "DragonSlayer.h"
 #include "Dragon.h"
+#include "Utility.h"
 
 //DragonSlayer::DragonSlayer
+DragonSlayer::DragonSlayer(std::string n,int hp, int armor) : Character(hp, armor, 4), name(n)
+{
```

Comment:
add space after `,` to make the code more readable
It's important to make the code as easy to read as possible, and that starts with consistency with the coding style
- [ ] revise